packages/components/src/components/header/header.component.ts

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable @typescript-eslint/unbound-method */
+import { classMap } from 'lit/directives/class-map.js';
+import { html } from 'lit/static-html.js';
+import { property } from 'lit/decorators.js';
+import type { CSSResultGroup } from 'lit';
+import SynergyElement from '../../internal/synergy-element.js';
+import { HasSlotController } from '../../internal/slot.js';
+import componentStyles from '../../styles/component.styles.js';
+import styles from './header.styles.js';
+import SynIcon from '../icon/icon.js';
+
+/**
+ * @summary The <syn-header /> element provides a generic application header
+ * that can be used to add applications name, toolbar and primary navigation.
+ *
+ * @documentation https://synergy-design-system.github.io/?path=/docs/components-syn-header--docs
+ * @status stable
+ * @since 1.8.0
+ *
+ * @slot - The label for the header. Will automatically be hidden on mobile.
+ * @slot logo - The logo that should be displayed. Will fall back to the SICK logo if not applied.
+ * @slot meta-navigation - Used to add various application toolbar icons.
+ *                     Best used with `<syn-icon-button />` and `<syn-drop-down />`
+ * @slot navigation - Used to add an optional horizontal navigation
+ *
+ * @csspart base - The component's base wrapper.
+ * @csspart content - The wrapper where most content items reside
+ * @csspart logo - The wrapper where the application logo resides in
+ * @csspart label - Wrapper of the application name label
+ * @csspart meta-navigation - Item that wraps the optional application menu
+ * @csspart navigation - Wrapper that holds the optional top navigation section
+ */
+export default class SynHeader extends SynergyElement {
+  static styles: CSSResultGroup = [
+    componentStyles,
+    styles,
+  ];
+
+  static dependencies = {
+    'syn-icon': SynIcon,
+  };
+
+  private readonly hasSlotController = new HasSlotController(this, '[default]', 'logo', 'label', 'meta-navigation', 'navigation');
+
+  /**
+   * The headers label. If you need to display HTML, use the `label` slot instead.
+   */
+  @property() label = '';
+
+  /**
+   * The logo-label attribute can be used as fallback description text
+   * in cases where the logo cannot be loaded.
+   * This only works for the `syn-icon` that gets slotted into the `navigation` as fallback.
+   * If custom content is provided in the `logo` slot,
+   * please take care that it is accessible by yourself!
+   */
+  @property({ attribute: 'logo-label' }) logoLabel = 'SICK Sensor Intelligence';
```

Comment:
This feels over-engineered to me and I don't see a reason why people should be able to override the label at this point if they don't change the log. Lots of "ifes". We have a syn-icon and the icon is a system icon, what should go wrong? If there is an issue it's a bug – but pre-fixing this with a prop feels useless.

Currently we have a little problem here: ATM it's only possible to add a link to these elements inside a slot? Therefore we definitely need the logo in the system icons, so people can slot it in combination with an `a href` or `button` whatever they need.

Reply:
Please discuss with @kirchsuSICKAG, but I don't see any reason for this overridable prop, as this even is ALWAYS the SICK logo.

Reply:
Thanks for the input. I see where you are coming from with this. When I created it I just thought about using the `label` prop as default, but it may also very well be the user is providing a `<slot />` for this, making it harder to get to the content. As we only need this when NOT providing a custom logo, I too see no reason for this to exist anymore, so opted to remove it.